### ISSUE TITLE
Internal error when trying to edit a project

### ISSUE THREAD

**Issue by Hexstream** [ultralisp/ultralisp#96]
I already successfully added most of my ready-to-use libraries to my `Hexstream/ready-to-use` ultralisp dist, but now when I click `edit` on [simple-guess](https://ultralisp.org/projects/Hexstream/simple-guess), I get "Oops, we could not complete your request because of an internal error.". I get that for [place-utils](https://ultralisp.org/projects/Hexstream/place-utils) also, so I think I might have triggered some sort of failure when adding my other projects.

**Comment by svetlyak40wt** [comment]
Here is what I've found in the log:

```
Returning 500 error to user ULTRALISP/SERVER::TRACEBACK: "Date/time: 2021-01-12-14:18!
An unhandled error condition has been signalled:
   An HTTP request to \"https://api.github.com/repos/Hexstream/simple-guess/branches\" returned 403 forbidden.

{\"message\":\"API rate limit exceeded for 104.248.211.197. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)\",\"documentation_url\":\"https://developer.github.com/v3/#rate-limiting\"}
```

When opening the edit form, Ultralisp fetches the branches list from the GitHub. And it hit the limit. It is about 60 requests per hour only for unauthenticated requests.

This was a temporary issue. I think, the best way to overcome this limitation would be to use current user's oauth token when doing such requests to the GitHub.

**Comment by svetlyak40wt** [comment]
@Hexstream please, try to edit projects now, probably there the rate limit was refreshed.

**Comment by Hexstream** [comment]
Yes, it works now and I successfully added all my ready-to-use libraries. Thank you!
